content/ja/project-management.md

```diff
@@ -89,7 +89,7 @@ menu:
 
 ・・・等、上記以外にも様々なプロジェクトを手懸けております。
 
-### PMアプローチ
+#### PMアプローチ
```

Comment:
@rick noted. but I thought it looks better when it changed smaller font.

2015/06/17 16:43$B!"(BRick Cogley <notifications@github.com> $B$N%a%C%;!<%8(B:

> In content/ja/project-management.md:
> 
> > @@ -89,7 +89,7 @@ menu:
> >  
> >  $B!&!&!&Ey!">e5-0J30$K$bMM!9$J%W%m%8%'%/%H$r<j7|$1$F$*$j$^$9!#(B
> >  
> > -### PM$B%"%W%m!<%A(B
> > +#### PM$B%"%W%m!<%A(B
> @eishikawa not sure if I need to use your name, for you to get the comment, so here is another one with your name.
> 
> $B!=(B
> Reply to this email directly or view it on GitHub.
>